doc/HACKING/ReleaseSeriesLifecycle.md

```diff
@@ -48,3 +50,49 @@ deprecations of individual versions.
    number from their approved versions list.
 
 7. Update the CoreTorReleases wiki page.
+
+8. Update the git/hooks
+
+9. Finally, make sure this document is up to date with our latest
+   process.
+
+10. Open a ticket (if there is not one already) for authorities to
+    start rejecting relays that are running that release series.
+    This ticket should be targeted for at least a month or two
+    after the series is officially EOL, unless there is an important
+    reason to un-list relays early.
+
+11. Open a ticket (if appropriate) for updates to the set of
+    required and recommended subprotocol versions.
+
```

Comment:
Let's also add an LTS end of life step for consensus method removal:
https://gitweb.torproject.org/torspec.git/tree/proposals/290-deprecate-consensus-methods.txt

Reply:
Let's also add this end of life step for all releases:
grep for references to obsolete releases (for example: 0.2.9 and 029), and update or remove them in:
* tor
* chutney
* sbws
* fallback-scripts

Reply:
I don't know if you want to add this step - it's not related to tor releases, but they are a convenient time to do it:

Update to the latest CI dependencies:
* OS images (Linux, macOS, and Windows)
* python versions (python and pypy, in chutney, sbws, and fallback-scripts)